### ISSUE TITLE
LVD: Leaving Kerbin SOI with 3000 m/s delta-v

### ISSUE THREAD

**Issue by Josef37** [Arrowstar/ksptot#16]
I just tested LVD today and got my craft to leave Kerbin's SOI with 3000 m/s delta-v.

Engine Isp is 278.36 s (sea level and vacuum), payload mass is 2 t and fuel mass is 4 t.
So delta v = 278.36 s * 9.81 m/s^2 * ln(6/2) = **3000 m/s**

The Wiki states as escape velocity over **3400 m/s** (https://wiki.kerbalspaceprogram.com/wiki/Kerbin).

I didn't have time to figure out why it happens, but I noticed that the _surface velocity is about an order of magnitude lower_ than it should be.

Here is the LVD file:
[LeaveKerbinSOI.zip](https://github.com/Arrowstar/ksptot/files/2566383/LeaveKerbinSOI.zip)

**Comment by Arrowstar** [comment]
Hey congrats you found a bug with my understanding of variable mass systems.  I've corrected it.  To test, I put myself at the surface of Kerbin in Mission Architect and added the escape velocity DV.  I got roughly what the wiki says, ~3400 m/s.

I then cranked up the engine thrust in LVD to something very high such that it would behave impulsively and matched the ISP from MA.  I got the same orbit out after the fix as I see in MA, so I match the impulsive assumption.  I did not before.  

I think this one can get closed, but I'll do some more testing tomorrow and double check.  Good find! :)

I attached my modified version of your LVD MAT file below if you want to play with it when I release LVD with the fix. :)

![image](https://user-images.githubusercontent.com/5885662/48297998-c6f6b480-e47a-11e8-9aa8-9aace6da3718.png)

![image](https://user-images.githubusercontent.com/5885662/48298002-d413a380-e47a-11e8-8948-ba57cbc5e54b.png)

[LeaveKerbinSOI.zip](https://github.com/Arrowstar/ksptot/files/2568156/LeaveKerbinSOI.zip)
